siphon/catalog.py

```diff
@@ -275,59 +272,6 @@ def __init__(self, service_node):
         self.services = services
 
 
-def basic_http_request(full_url, return_response=False):
-    r"""
```

Comment:
Well that's embarrassing...what kind of doc string did I write in there! I'll tell you - the wrong one!